### PR TITLE
V1.8

### PR DESCRIPTION
**Pull request by transistive**
Hi all,

I tried to integrate Neo Eloquent to work with newer versions of Laravel. I don't have a lot of time, so I started with the most relevant one: Laravel 8. It appears to work, minus the bugs still existing with ids.

Hope all is well,

Ghlen

### CODE REVIEW

File: src/Eloquent/Model.php [hunk 1]
```diff
@@ -3728,9 +3728,18 @@ public function getQueueableConnection()
...
 
+    public function getQueueableRelations()
+    {
+        // TODO: Implement getQueueableRelations() method.
+    }
+
+    public function resolveChildRouteBinding($childType, $value, $field)
+    {
+        // TODO: Implement resolveChildRouteBinding() method.
+    }
```
Comment (Mulkave) [inline src/Eloquent/Model.php:3744]: not sure whether we should provide implementations for these, or what features they implement.

Comment (transistive) [reply src/Eloquent/Model.php:3744]: I have not a clue, but they are new model methods that had to be there. We might want to throw BadMethodCall Exceptions to defend against unintended usage until we figure out what they are used for?

Comment (Mulkave) [reply src/Eloquent/Model.php:3744]: Oh, also there are merge conflicts in `composer.json`, i suppose we should just override with the version in this PR>

**Comment by kirkbushell** [comment]
This appears to be busted, seeing the following error:

> Declaration of Vinelab\NeoEloquent\ConnectionAdapter::selectOne($query, $bindings = []) must be compatible with Illuminate\Database\Connection::selectOne($query, $bindings = [], $useReadPdo = true)

This doesn't appear to have been tested very well, additionally the documentation for installation and configuration is now very outdated.

Finally, this is a very breakable change, and so probably should have been a 2.x release, imho.

**Comment by kirkbushell** [comment]
I stepped through this, and I'm not sure how this release is meant to work - there are so many issues with the API in Laravel 8.

**Pull request merged**